### DISCUSSION TITLE
How to use/convert .fla files with HaxeFlixel?

### DISCUSSION THREAD

**Discussion by Maniues** [HaxeFlixel/flixel#2391] [Q&A]
How to use .fla files with HaxeFlixel? I want to convert .fla files to swf or mp4/ogv without Adobe Animate.

**Comment by RichardBray** [comment]
I may be wrong but I don't know if it's possible. Flixel sits on top of [OpenFL](https://www.openfl.org/) so if it can be done with that it can be done with Flixel. But .fla files don't work natively on the web and may be a struggle for desktop targets. I don't know if [Lime](https://lime.software/) is clever enough to do that sort of audio conversion. sh🤷

**Comment by TheHENOOB** [comment]
What are you trying to do isn't possible on haxe and this task only relies on using animate or flash

other info:

- if you want to convert .swf to .mp4 use [swivel](https://www.newgrounds.com/wiki/creator-resources/animation-resources/swivel-flash-to-video)
- Haxe/OpenFL has the ability to hack .swf files and bring symbols and assets to CPP and HTML5 but it is limited (see https://github.com/openfl/swf)

**Reply by Maniues** [reply]
swf is not .fla

**Reply by TheHENOOB** [reply]
I know, as i said

> what are you trying to do isn't possible on haxe and this task only relies on using animate or flash

**Reply by Maniues** [reply]
Oh, sorry
